sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -239,15 +239,15 @@ NULL
 -- !query 29
 select -interval '-1 month 1 day -1 second'
 -- !query 29 schema
-struct<1 months -1 days 1 seconds:interval>
+struct<(- -1 months 1 days -1 seconds):interval>
 -- !query 29 output
 1 months -1 days 1 seconds
 
 
 -- !query 30
 select -interval -1 month 1 day -1 second
 -- !query 30 schema
-struct<1 months -1 days 1 seconds:interval>
+struct<(- -1 months 1 days -1 seconds):interval>
```

Comment:
the schema looks weird here

Reply:
will be fixed by https://github.com/apache/spark/pull/26579